fastlane/Fastfile

```diff
@@ -161,6 +161,27 @@ platform :android do
     )
   end
 
+  desc "Build purchases module integration tests pointing to production"
+  lane :run_backend_integration_tests do |options|
+    constants_path = './common/src/test/java/com/revenuecat/purchases/backend_integration_tests/Constants.kt'
+    replace_text_in_files(
+      previous_text: 'REVENUECAT_API_KEY',
+      new_text: ENV['REVENUECAT_API_KEY'],
+      paths_of_files_to_update: [constants_path]
+    )
+    replace_text_in_files(
+      previous_text: 'LOAD_SHEDDER_API_KEY',
```

Comment:
Should this be `LOAD_SHEDDER_REVENUECAT_API_KEY`?

Reply:
So I used `LOAD_SHEDDER_API_KEY` in the `Constants.kt` file to avoid accidentally changing it in the previous replace, where we replace `REVENUECAT_API_KEY`.  We could use patterns to avoid that problem though I guess, but this works for now.